### DISCUSSION TITLE
Could you enable the sponsor button on this repo?

### DISCUSSION THREAD

**Discussion by fenske** [stedolan/jq#3046] [Ideas]
I'm using `jq` for my SaaS and I thought it might be a not that bad idea to wire a few bucks to this project via GitHub sponsors. 

But it seems to be disabled on the repo.

Would you be ok with turning it on?

**Comment by demarcusw** [comment]
+1. surprised there's no financial support option, so many orgs depend on jq!

**Comment by ffmmjj** [comment]
This :arrow_up:

**Comment by bresciamattia** [comment]
@stedolan any chance you can handle this?

**Comment by mrienstra** [comment]
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository

**Comment by nicowilliams** [comment]
So far all maintainers have been volunteers, and @stedolan wrote `jq` w/o expectation of direct compensation.  I think we the maintainers get indirect compensation as it is (e.g., mainly in making us more appealing as potential employees for better-paying jobs), and while I am not interested in _direct_ compensation for work on `jq`,  maybe others are, and all we'd have to do is figure out how to deal with any contributions.  We have a discord where maintainers can discuss this.
